src/connector.rs

```diff
@@ -2,7 +2,7 @@ use ct_logs;
 use futures::{Future, Poll};
 use hyper::client::connect::{self, Connect};
 use hyper::client::HttpConnector;
-use rustls::ClientConfig;
+use rustls::{ClientConfig, ClientSession};
```

Comment:
I think this was supposed to be `Session` instead.

Reply:
Woops, yep! All fixed.